createBoard.json

```diff
@@ -0,0 +1,13 @@
+{
+    "selectOrg" : ".el-input input",
+    "dropdownOrgOption" : "li.vs-c-select-dropdown__item",
+    "page3" : ".vs-c-dot-pagination > li:nth-of-type(3)",
```

Comment:
What are these?

Reply:
@bdimitrovski 
The "page3"  (to page5) is for pagination, so we can check which one is active during the creation process.
The "dropdownOrgOption" is for one Organization I have created from the dropdown menu.
The "selectOrg" is the input field that should be used for entering the Organization name OR for expanding the dropdown menu.